### ISSUE TITLE
timeclock for schedule

### ISSUE THREAD

**Issue by vesnick** [rakshasa/rtorrent#1049]
How to set up a schedule for each job by timeclock?

Need line command to define stop | start | resume | pause of all jobs in one specific schedule_watch

Something like
stop_tied
start_tied
pause_tied
resume_tied
close_tied

And use it somehow:

schedule2 = close_sdb_linux,18:00:00,24:00:00,"load.normal=rtorrents/sdb_linux/*.torrent,d.multicall,d.directory.set=/mnt/sdb/Linux,close_tied="
schedule2 = start_sdb_linux,06:00:00,24:00:00,"load.normal=rtorrents/sdb_linux/*.torrent,d.multicall,d.directory.set=/mnt/sdb/Linux,start_tied="

I am sure there is such an opportunity, because in all other respects your client is perfect

**Comment by pyroscope** [comment]
First, you need to describe in plain English sentences what you actually want. And use Reddit for support, not issues.

**Comment by vesnick** [comment]
> First, you need to describe in plain English sentences what you actually want. And use Reddit for support, not issues.

run/stop at hours tied
